package.json

```diff
@@ -14,14 +14,17 @@
       "react-dom": "^18.2.0"
    },
    "devDependencies": {
+      "@playwright/test": "^1.44.0",
       "@types/react": "^18.2.74",
       "@types/react-dom": "^18.2.24",
       "@vitejs/plugin-react": "^4.2.1",
```

Comment:
In Codux we are only testing on Chromium as much as I know so no need to the whole package of playwright with all the data for all browsers.
So lets install only the ```playwright-chromium``` package.